wazimap_ng/datasets/admin/forms/dataset_admin_form.py

```diff
@@ -34,9 +33,4 @@ def __init__(self, *args, **kwargs):
                 ).first()
 
             if filtered_profile:
-                self.fields["profile"].initial = filtered_profile
-
-                for hierarchy in hierarchies:
-                    hp = hierarchy.profile_set.all()
-                    if hp.count() == 1 and hp.first() == filtered_profile:
-                        self.fields["geography_hierarchy"].initial = hierarchy
```

Comment:
looks like this is how a relevant initial hierarchy used to be selected. Perhaps this is what we want for version too?